src/includes/configuration/before-send-fingerprint/android.mdx

```diff
@@ -5,7 +5,7 @@ import java.util.Arrays;
 
 SentryAndroid.init(this, options -> {
   options.setBeforeSend((event, hint) -> {
-    if (event.getThrowable() instanceof SQLiteException) {
+    if (event.getOriginThrowable() instanceof SQLiteException) {
```

Comment:
Doesn't the getter of throwable already returns the original throwable?

Reply:
no

Reply:
I'm just curious. Why did we rename this?

Reply:
see the linked Dart issue, same thing.